### DISCUSSION TITLE
what are "banks" in sprite editor?

### DISCUSSION THREAD

**Discussion by atesin** [nesbox/TIC-80#2197] [Q&A]
hi....   what are "bank0" and "bank1" next to palette, in sprite editor advanced mode?? 

![image](https://github.com/nesbox/TIC-80/assets/8086106/6d857709-ea5c-4103-88ab-726e06c9c20d)

when i select "bank1" every color turns black

... what are for?... how to use it? ...  the documentation doesn't even mention it

**Comment by joshgoebel** [comment]
https://github.com/nesbox/TIC-80/wiki/vbank

**Comment by atesin** [comment]
so... that mean, we can just switch to bank1 and work the same as bank0, and we got another exclusive set of colors+palette+sprites+maps just for OVR() function use??  ...  that would mean that we could actually have 32 color palette with composite sprites!

**Reply by joshgoebel** [reply]
No, the video banks contain just 32kb (VRAM itself + registers)... so yes there is another palette, but not additional sprites or maps or such things.  Also don't get the two VRAM banks confused with the 8 cartridge banks.

Yes, you can have 31 color palette with sprite composition (you need to save a color for transparency).

**Reply by atesin** [reply]
hi.....  i made a quick test of what you said .. i learned a lot

https://tic80.com/play?cart=3361

( OVR() function is somewhat weird,... has to draw every frame, F7 screenshot resulted different, has some other oddities, etc)
